flocx_market/api/app.py

```diff
@@ -1,25 +1,14 @@
-import os
 from flask import Flask
-from flask import Blueprint
-from flask import jsonify
-import flocx_market.conf
-CONF = flocx_market.conf.CONF
-flocx_market_api_bp = Blueprint("flocx_market_api_bp", __name__)
+from flask_restful import Api
 
-
-@flocx_market_api_bp.route('/')
-def index():
-    flocx_market_url = CONF.api.host_ip + ':' + str(CONF.api.port)
-    version = {"versions":
-               {"values": [{"status": "in progress",
-                            "updated": "2019-07-02T00:00:00Z",
-                            "media-types": [{"base": "application/json", "name": "flocx-market"}],
-                            "links": [{"href": flocx_market_url, "rel": "self"}]}]
-                }}
-    return jsonify(version)
+from flocx_market.api.offer import Offer, OfferList
 
 
 def create_app(app_name):
     app = Flask(app_name)
-    app.register_blueprint(flocx_market_api_bp)
+    app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+pymysql://flocx_market:qwerty123@127.0.0.1:3307/flocx_market'
+    app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+    app.config['PROPAGATE_EXCEPTIONS'] = True
```

Comment:
Two comments:

* These values should be retrieved using oslo config - CONF.database.connection, and CONF.flask.SQLALCHEMY_TRACK_MODIFICATIONS, etc - ask @DanNiESh or @ljmcgann if you want help with that

* The database configuration should not be done here, but somewhere in db/sqlalchemy/. Remember - the REST API is not the only route into working with these objects, and we don't want to duplicate this in multiple places. Take a look at https://github.com/openstack/ironic/blob/master/ironic/db/sqlalchemy/models.py for reference.